Firmware/tmc2130.cpp

```diff
@@ -867,13 +867,10 @@ void tmc2130_get_wave(uint8_t axis, uint8_t* data, FILE* stream)
 		uint32_t val = tmc2130_rd_MSCURACT(axis);
 		uint16_t mscnt = tmc2130_rd_MSCNT(axis);
 		int curA = (val & 0xff) | ((val << 7) & 0x8000);
-		if (stream)
-		{
-			if (mscnt == i)
-				fprintf_P(stream, PSTR("%d\t%d\n"), i, curA);
-			else //TODO - remove this check
-				fprintf_P(stream, PSTR("!! (i=%d MSCNT=%d)\n"), i, mscnt);
-		}
+		if (mscnt == i)
+			printf_P(PSTR("%d\t%d\n"), i, curA);
+		else //TODO - remove this check
+			printf_P(PSTR("!! (i=%d MSCNT=%d)\n"), i, mscnt);
```

Comment:
Please change `!!` to single `!` to prevent OctoPrint to ERROR out

Reply:
Done